### ISSUE TITLE
Allow us to put a cooldown on 1 of multiple events instead of all of them!

### ISSUE THREAD

**Issue by ItzASecret** [PluginBugs/Issues-ItemsAdder#1358]
### Terms

- [X] I'm using the very latest version of ItemsAdder and its dependencies.
- [X] I already searched on this [Github page](https://github.com/PluginBugs/Issues-ItemsAdder/issues) to check if the same suggestion was already reported.
- [X] I already searched on the [plugin wiki](https://itemsadder.devs.beer/) to know if the feature is already implemented.
- [X] I already searched on the **Discord** server to know if anyone already has a solution for this or if the feature was already suggested.

### Discord tag (optional)

ItzASecretBoi0#1300

### Describe the solution you'd like

Okay, I would like to be able to put a cooldown on one action instead of all of them. I am making a hammer right now, and when you right click it it does damage to enemies around you. The problem comes in because I also have a action that make you have slowness when holding the hammer, however they will not give slowness or remove it if they have a cooldown active because there is a cooldown on the action the gives and removes slowness. So instead I would like to put the cooldown on the right clicking action ONLY. 

tldr put cooldown on specific action instead of all of them

### Is your feature request related to a problem?

My hammer item doesn't work as I wished, because of the cooldown you either get stuck with slowness forever by holding it or you get to hold the OP hammer without slowness

### Describe alternatives you've considered

None I asked discord sircraftypants told me to suggest on github

### Additional context

Your pretty neat LoneDev! Thanks for making this plugin, best purchace ive made!

**Comment by LoneDev6** [comment]
> Your pretty neat LoneDev! Thanks for making this plugin, best purchace ive made!

Thanks a lot!

Currently it's not really easy to implement without confusing the player. The player might think their item is bugged if some actions don't trigger, don't you think?

**Comment by ItzASecret** [comment]
Yeah, I understand it could be confusing. Just an idea I had based on my situation. If it isnt plausible the player then I see why it wouldnt make it.

**Comment by LoneDev6** [comment]
This kind of customization might be worth to be implemented as a plugin, I think the complexity of your items require more customization which can be achieved using Java.

**Comment by ItzASecret** [comment]
I cant code haha, I just started my server this year! I'm beginner status. I just understand Itemsadder to a decent extent.

**Comment by LoneDev6** [comment]
I'm currently thinking on a clean way to let the user know about the cooldowns.
Do you think a lore would be enough?

I mean, you write down the item features and cooldowns by yourself so that the player will know about the cooldowns.
